pkg/scheduler/core/generic_scheduler.go

```diff
@@ -778,10 +722,6 @@ func (g *genericScheduler) prioritizeNodes(
 
 	for i := range nodes {
 		result = append(result, framework.NodeScore{Name: nodes[i].Name, Score: 0})
-		for j := range g.prioritizers {
```

Comment:
```RunScorePlugins``` should return the result in the desired format rather than doing the copy here, but this can be done in a follow up PR.